pages/establishment-address.js

```diff
@@ -21,7 +21,9 @@ const EstablishmentAddress = (props) => (
       onHandleErrorClick={OnHandleErrorClick}
     />
     <Heading as="h1" size="LARGE">
-      {props.t("Establishment address")}
```

Comment:
they suggested to use the h1 heading as the label for the input field since it's the only content on the page. CHeck here for an example: https://design-system.service.gov.uk/patterns/question-pages/

The 'for' attribute on the label tag of the h1 heading links it back to the input field as it matches the input field's id

Reply:
Ah okay I see, all good. In that case I'd just try and copy their example exactly and include the class="govuk-label ..." stuff too then. Just reduces the chance for it to be brought up on a future report.